src/appleseed/foundation/array/arrayallocator.h

```diff
@@ -30,7 +30,7 @@
 
 // appleseed.foundation headers.
 #include "foundation/math/scalar.h"
-#include "foundation/utility/memory.h"
+#include "foundation/memory/memory.h"
```

Comment:
Maybe we could rename `memory.h`?

Reply:
I prefer to leave it like this. having memory.h and memory/ with some other headers looks a bit arbitrary to me.

Reply:
I'm not sure to understand your point :) Mine was: now that we have a `memory` subfolder, `memory.h` isn't very descriptive anymore and maybe we could rename it. Or not. Just wondering.